src/runtime/components/forms/Form.vue

```diff
@@ -220,6 +222,35 @@ function isZodSchema(schema: any): schema is ZodSchema {
   return schema.parse !== undefined
 }
 
+export function isStandardSchema(schema: any): schema is StandardSchemaV1 {
+  return '~standard' in schema
+}
+
+export async function validateStandarSchema(
+  state: any,
+  schema: StandardSchemaV1
+): Promise<ValidateReturnSchema<typeof state>> {
+  const result = await schema['~standard'].validate(state)
```

Comment:
Any idea why [this is different in v3](https://github.com/nuxt/ui/blob/f3f4edae2862aebe5de7639668f2505ffdd3718d/src/runtime/utils/form.ts#L50)?

The current version breaks Valibot schemas, this would fix that, but I can't tell if Valibot is wrong, or the Nuxt implementation is wrong?